submitit/slurm/_sbatch_test_record.txt

```diff
@@ -10,7 +10,7 @@
 #SBATCH --open-mode=append
 #SBATCH --output=/tmp/%j_0_log.out
 #SBATCH --partition=learnfair
-#SBATCH --signal=USR1@90
+#SBATCH --signal=TERM@90
```

Comment:
Is this change necessary now? :s don't we get it anyway?

Reply:
we always get it, but we want to be able to control the delay.